addons/policies/bootstrap_entity_policies.json

```diff
@@ -2710,6 +2710,7 @@
                     "entity-type:Process",
                     "entity-type:Connection",
                     "entity-type:ProcessExecution",
+                    "entity-type:File",
```

Comment:
Are you sure we should deny CRUD explicitly $guest role?

Reply:
I saw Heracles PR, it is not denied explicitly - https://github.com/atlanhq/heracles/pull/1499/files